dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/config/SpringConfigBackendDatabase.java

```diff
@@ -59,6 +60,13 @@
 
   @Autowired ObjectMapper objectMapper;
 
+  public static final ThreadLocal<URL> iiifImageBaseUrl = new ThreadLocal<>();
```

Comment:
WArum muss das ein `ThreadLocal` sein?

Reply:
Falls es nur für Injection nach Flyway gebraucht wird, hierfür bitte besser den Migration Script Placeholder Support in Flyway/Spring Boot verwenden:

https://flywaydb.org/documentation/placeholders
https://stackoverflow.com/questions/49161522/spring-boot-flyway-migration-placeholder

Reply:
Ist erledigt.